core/cli/node/filplus.hpp

```diff
@@ -0,0 +1,190 @@
+      if (data_cap < allowness)
+        throw CliError(
+            "cannot allow more allowance than notary data cap: {} < {}",
+            data_cap,
+            allowness);
+
+      auto encoded_params = cliTry(codec::cbor::encode(
+          vm::actor::builtin::v0::verified_registry::AddVerifiedClient::Params{
+              target, allowness}));
+      SignedMessage signed_message = cliTry(api->MpoolPushMessage(
+          {kVerifiedRegistryAddress,
+           *args.from,
+           {},
+           0,
+           0,
+           0,
+           vm::actor::builtin::v0::verified_registry::AddVerifiedClient::Number,
+           encoded_params},
+          api::kPushNoSpec));
+
+      fmt::print("message sent, now waiting on cid: {}\n",
+                 signed_message.getCid());
+      const MsgWait message_wait = cliTry(api->StateWaitMsg(
+          signed_message.getCid(), kMessageConfidence, kLookBack, false));
+      if (message_wait.receipt.exit_code != VMExitCode::kOk)
+        throw CliError("failed to add verified client");
+      fmt::print("Client {} was added successfully!\n", target);
+    }
+  };
+
+  struct Node_client_listNotaries : Empty {
+    CLI_RUN() {
+      const Node::Api api{argm};
+
+      const Actor actor =
+          cliTry(api->StateGetActor(kVerifiedRegistryAddress, TipsetKey()),
+                 "Getting VerifierActor");
+      auto ipfs = std::make_shared<ApiIpfsDatastore>(api.api);
+      const NetworkVersion version = cliTry(
+          api->StateNetworkVersion(TipsetKey()), "Getting Chain Version...");
+      ipfs->actor_version = actorVersion(version);
+
+      const auto state =
+          cliTry(getCbor<VerifiedRegistryActorStatePtr>(ipfs, actor.head));
+
+      cliTry(state->verifiers.visit(
+          [=](auto &key, auto &value) -> outcome::result<void> {
+            fmt::print("{}: {}\n", key, value);
+            return outcome::success();
+          }));
+    }
+  };
+
+  // Only for local networks, wont work in main-net
+  struct Node_client_addVerifier {
+    struct Args {
+      CLI_OPTIONAL("from",
+                   "specifies the address of notary to send message from",
+                   Address)
+      from;
+      CLI_OPTS() {
+        Opts opts;
+        from(opts);
+        return opts;
+      }
+    };
+
+    CLI_RUN() {
+      const Node::Api api{argm};
+
+      const Actor actor =
+          cliTry(api->StateGetActor(kVerifiedRegistryAddress, TipsetKey{}));
+      auto ipfs = std::make_shared<ApiIpfsDatastore>(api.api);
+      const NetworkVersion network =
+          cliTry(api->StateNetworkVersion(TipsetKey{}));
+      ipfs->actor_version = actorVersion(network);
+
+      auto state =
+          cliTry(getCbor<VerifiedRegistryActorStatePtr>(ipfs, actor.head));
+
+      SignedMessage signed_message0 = cliTry(api->MpoolPushMessage(
+          vm::message::UnsignedMessage(
+              kVerifiedRegistryAddress, state->root_key, 0, 0, 0, 0, 0, {}),
```

Comment:
Did you want to create account (`*opts.from`, not `kVerifiedRegistryAddress`)?  
Maybe remove that, because user can create account with other cli method.  
Otherwise need option to choose sender for account creation

Reply:
Fixed